Resources/Prototypes/Recipes/Reactions/medicine.yml

```diff
@@ -672,3 +672,37 @@
       amount: 1
   products:
     SalineGlucose: 3
+
+- type: reaction
+  id: Salicyclic
```

Comment:
Either this needs to be changed.

Reply:
it's definitely something wrong i did with this one, idk what's happening with it, i'll try to change the recipe and ID